### DISCUSSION TITLE
Default list of blacklisted apps

### DISCUSSION THREAD

**Discussion by derivativeoflog7** [MicaForEveryone/MicaForEveryone#122] [Ideas]
Would it be feasible to create a default blacklist for apps that are known to break with MFE enabled? (eg: games that have picture or texture issues)

**Comment by dongle-the-gadget** [comment]
We don't blacklist apps in the source code itself, but rather through the default configuration file.

**Reply by derivativeoflog7** [reply]
> We don't blacklist apps in the source code itself, but rather through the default configuration file.

Is there a place to report incompatible apps?

**Reply by dongle-the-gadget** [reply]
There are none currently. Usually, people would post it in Issues to be marked as third-party.
